runtime/src/invoke_dpe.rs

```diff
@@ -121,19 +122,25 @@ impl InvokeDpeCmd {
         locality: u32,
         dpe: &DpeInstance,
     ) -> CaliptraResult<()> {
-        let active_pl0_dpe_context_count = dpe
-            .count_active_contexts_in_locality(pl0_pauser)
+        let non_inactive_context_pred = |c: &Context| c.state != ContextState::Inactive;
+        let non_inactive_contexts_in_locality_pred =
+            |c: &Context| non_inactive_context_pred(c) && c.locality == pl0_pauser;
+        let non_inactive_pl0_dpe_context_count = dpe
+            .count_contexts(non_inactive_contexts_in_locality_pred)
             .map_err(|_| CaliptraError::RUNTIME_INTERNAL)?;
-        let active_pl1_dpe_context_count = dpe
-            .count_active_contexts()
+        // the number of non-inactive pl1 dpe contexts is the total number of non-inactive contexts
+        // minus the number of non-inactive pl0 contexts, since a context can only be activated
+        // from pl0 or from pl1.
+        let non_inactive_pl1_dpe_context_count = dpe
```

Comment:
nit: Maybe replace `non_inactive` with `used`. Can add a comment that `used` includes Active and Retired contexts

Reply:
Done

Reply:
I don't see this updated. Did it get missed in a commit?

Reply:
strange, could you refresh? its updated on my end

Reply:
oh did you mean updating the variable names as well to say used instead of non_inactive?

Reply:
Updated the variable names and error codes as well from non_inactive -> used.